src/platform/plugins/shared/dashboard/public/dashboard_api/track_panel.ts

```diff
@@ -9,6 +9,8 @@
 
 import { BehaviorSubject, Subject } from 'rxjs';
 
+export const highlightAnimationDuration = 3000;
```

Comment:
since this value is related to both animation and class removal timeout, I'd keep it in variable.